sign_oca/tests/test_sign_portal.py

```diff
@@ -47,6 +47,7 @@ def setUp(self):
         )
 
     def test_portal(self):
+        self.authenticate("portal", "portal")
```

Comment:
Why do you need to authenticate? Previously this was not needed :thinking:

Reply:
Without this change, when I run the test locally I get the error:

`requests.exceptions.HTTPError: 404 Client Error: NOT FOUND for url: http://127.0.0.1:8069/sign_oca/document/8/xxxxxxx`

Other tests of other modules that use `self.url_open()` also set `self.authenticate()` before.